### ISSUE TITLE
Bigger Brains: Notify when new courses are released

### ISSUE THREAD

**Issue by PreciseNZ** [DeskDirector/UserVoice#310]
Would like to be able to see when a new course is released in order to pass details onto end customers.  This will encourage returning traffic, feeding the consumption cycle.

**Comment by Lowell-Syncrety** [comment]
Yes! I think I had a ticket requesting this. I know new stuff like Teams has been added but I'd like to send out an email to customers both reminding them of this training and letting them know of new stuff. DD is always asking us for ideas on better ways to increase customer engagement and usage of DD products and this is a super-easy win for that.
